frontend/src/components/atoms/elements/Text.tsx

```diff
@@ -1,17 +1,23 @@
+import { useRecoilValue } from 'recoil';
+
 import Box from '@mui/material/Box';
 
 import { Markdown } from 'components/molecules/Markdown';
 
 import { useFetch } from 'hooks/useFetch';
 
 import { type ITextElement } from 'client-types/';
+import { projectSettingsState } from '@chainlit/app/src/state/project';
```

Comment:
You can use a direct import because this file is already within the `@chainlit/app` package.

```suggestion
import { projectSettingsState } from 'state/project';
```

Reply:
done